For Advanced/020.md

```diff
@@ -0,0 +1,42 @@
+# Elementary ID
+
+Many of the `selenium.` commands allow to select an element on a page in order to click it, get or set its attribute, or perform other actions. They use the `search` and the `by` arguments to pinpoint the required element: you provide the phrase to search for an element after the `search` argument and the element selector after the `by` argument.
+
+Open [Google](https://google.com) in your browser. Now, right-click the **Sign In** button in the top-right corner of the page and select `Inspect` (Chrome) or `Inspect Element` (Firefox, IE) from the context menu. A new pane will be displayed on the right side (Chrome) or at the bottom (Firefox, IE) of the browser, revealing the code behind the page. Take a look at it — specifically, at the selected line of the code:
+
+```xml
+<a class="gb_Hd gb_1 gb_kb" id="gb_70" href="https://accounts.google.com/ServiceLogin?hl=en&amp;passive=true&amp;continue=https://www.google.com/" target="_top">Sign in</a>
+```
+
+This is the full description of an element in XML: the **Sign In** button on the Google page. Of course, it will differ if other language than English is set for the browser.
```

Comment:
html

Reply:
```suggestion
This is the full description of an element in HTML: the **Sign In** button on the Google page. Of course, it will differ if other language than English is set for the browser.
```